CGPs/0023.md

```diff
@@ -10,43 +10,42 @@
+  - Data: first argument is the StableTokenEUR address and the second one the address of the oracle to be added, same as in (1)
   - Value: 100e18  (note the value field of the proposal it's actually `0`, as this was implemented as a token transfer)
 3. Freezing StableTokenEUR
   - Destination: Freezer.freeze
-  - Data: StableTokenEUR (# TODO add address)
+  - Data: StableTokenEUR (`0xD8763CBa276a3738E6DE85b4b3bF5FDed6D6cA73`)
   - Value: 0
 4. Freezing ExchangeEUR
   - Destination: Freezer.freeze
-  - Data: ExchangeEUR (# TODO add address)
+  - Data: ExchangeEUR (`0xE383394B913d7302c49F794C7d3243c429d53D1d`)
   - Value: 0
 ## Verification
 
 Can be left as “TODO” until the proposal is made.
 
 ## Risks
 
-- Adding new oracles the network will be more congested, which is not a problem at this stage but could become one as new stable assets and other projects deploy new oracles.
--  Oracle rate risk: In case of imprecise CELO/USD oracle rates and/or manipulated CELO/USD market rates, users could exchange cUSD or CELO with the reserve at a price that does not reflect the current market valuation. This could negatively affect the reserve collateralization ratio and therefore long-term cUSD stability in two ways: First, a user could mint X cUSD while providing less than X USD worth of reserve collateral and second, a user could redeem Y cUSD for more than Y USD worth of reserve collateral. The major mitigant of this risk is the initial reduction of the reserveFraction parameter as explained in the overview section which comes at the cost of the risks described further below. Additional mitigants include oracle client features like a circuit breaker protecting against unreasonably large price movements. (Same as [CGP #0007](https://github.com/celo-org/celo-proposals/blob/master/CGPs/0007.md))
+- Adding new oracles will add load to the network, which is not a problem at this stage but could become one as new stable assets and other projects deploy new oracles and network adoption grows.
+- Oracle rate risk: In case of imprecise CELO/USD oracle rates and/or manipulated CELO/USD market rates, users could exchange cUSD or CELO with the reserve at a price that does not reflect the current market valuation. This could negatively affect the reserve collateralization ratio and therefore long-term cUSD stability in two ways: First, a user could mint X cUSD while providing less than X USD worth of reserve collateral and second, a user could redeem Y cUSD for more than Y USD worth of reserve collateral. The major mitigant of this risk is the initial reduction of the reserveFraction parameter as explained in the overview section which comes at the cost of the risks described further below. Additional mitigants include oracle client features like a circuit breaker protecting against unreasonably large price movements. (Same as [CGP #0007](https://github.com/celo-org/celo-proposals/blob/master/CGPs/0007.md))
```

Comment:
```suggestion
- Oracle rate risk: In case of imprecise CELO/EUR oracle rates and/or manipulated CELO/EUR market rates, users could exchange cEUR or CELO with the reserve at a price that does not reflect the current market valuation. This could negatively affect the reserve collateralization ratio and therefore long-term cEUR stability in two ways: First, a user could mint X cEUR while providing less than X EUR worth of reserve collateral and second, a user could redeem Y cEUR for more than Y EUR worth of reserve collateral. The major mitigant of this risk is the initial reduction of the reserveFraction parameter as explained in the overview section which comes at the cost of the risks described further below. Additional mitigants include oracle client features like a circuit breaker protecting against unreasonably large price movements. (Same as [CGP #0007](https://github.com/celo-org/celo-proposals/blob/master/CGPs/0007.md))
```